packages/local_auth/local_auth_darwin/darwin/Tests/FLALocalAuthPluginTests.swift

```diff
@@ -15,547 +16,509 @@ import XCTest
 // Set a long timeout to avoid flake due to slow CI.
 private let timeout: TimeInterval = 30.0
 
-/// A context factory that returns preset contexts.
-final class StubAuthContextFactory: NSObject, FLADAuthContextFactory {
-
-  var contexts: [FLADAuthContext]
-  init(contexts: [FLADAuthContext]) {
-    self.contexts = contexts
-  }
-
-  func createAuthContext() -> FLADAuthContext {
-    XCTAssert(self.contexts.count > 0, "Insufficient test contexts provided")
-    return self.contexts.removeFirst()
-  }
-}
-
-final class StubViewProvider: NSObject, FLAViewProvider {
-  #if os(macOS)
-    var view: NSView
-    var window: NSWindow
-    override init() {
-      self.window = NSWindow()
-      self.view = NSView()
-      self.window.contentView = self.view
-    }
-  #endif
-}
-
-#if os(macOS)
-  final class TestAlert: NSObject, FLANSAlert {
-    var messageText: String = ""
-    var buttons: [String] = []
-    var presentingWindow: NSWindow?
-
-    func addButton(withTitle title: String) -> NSButton {
-      buttons.append(title)
-      return NSButton()  // The return value is not used by the plugin.
-    }
-
-    func beginSheetModal(for sheetWindow: NSWindow) async -> NSApplication.ModalResponse {
-      presentingWindow = sheetWindow
-      return NSApplication.ModalResponse.OK
-    }
-  }
-#else
-  final class TestAlertController: NSObject, FLAUIAlertController {
-    var actions: [UIAlertAction] = []
-    var presented = false
-    var presentingViewController: UIViewController?
-
-    func add(_ action: UIAlertAction) {
-      actions.append(action)
-    }
-
-    func present(
-      on presentingViewController: UIViewController, animated flag: Bool,
-      completion: (() -> Void)? = nil
-    ) {
-      presented = true
-      self.presentingViewController = presentingViewController
-    }
-  }
-#endif
-
-final class StubAlertFactory: NSObject, FLADAlertFactory {
-  #if os(macOS)
-    var alert: TestAlert = TestAlert()
-  #else
-    var alertController: TestAlertController = TestAlertController()
-  #endif
-
-  #if os(macOS)
-    func createAlert() -> FLANSAlert {
-      return self.alert
-    }
-  #else
-    func createAlertController(
-      withTitle title: String?, message: String?, preferredStyle: UIAlertController.Style
-    ) -> FLAUIAlertController {
-      return self.alertController
-    }
-  #endif
-}
-
-final class StubAuthContext: NSObject, FLADAuthContext {
-  /// Whether calls to this stub are expected to be for biometric authentication.
-  ///
-  /// While this object could be set up to return different values for different policies, in
-  /// practice only one policy is needed by any given test, so this just allows asserting that the
-  /// code is calling with the intended policy.
```

Comment:
Why were all of the comments removed from this class?

Reply:
I actually forgot to add it back when I switched to the protocol.